dbt_af/parser/dbt_profiles.py

```diff
@@ -9,7 +9,7 @@
 
 
 class Target(BaseModel):
-    target_type: Literal['postgres', 'snowflake', 'bigquery', 'redshift', 'databricks'] = Field(alias='type')
+    target_type: str = Field(alias='type')
```

Comment:
`dbt-af` brings special target called `kubernetes`, so now it's parsed as a general target. You need to change order of type in `Profile` in this file:

```python
class Profile(BaseModel):
    ...
    outputs: dict[str, KubernetesTarget | Target]
```

Reply:
fixed